app/javascript/react/components/PizzasIndexContainer.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from "react"
+
+const PizzasIndexContainer = (props) => {
+  const [pizzas, setPizzas] = useState([])
+
+  const fetchPizzas = async () => {
+    const response = await fetch("/api/v1/pizzas")
+    const parsedPizzas = await response.json()
+    setPizzas(parsedPizzas)
+  }
+
+  useEffect(() => {
+    fetchPizzas()
+  }, [])
+
+  let pizzaTiles = pizzas.map((pizza) => {
+    return(
+      <li key={pizza.id}>
+        {pizza.brand}
+      </li>
+    )
+  })
+
+  return(
+    <div>
+      <h3> Pizzas Index </h3>
+      {pizzaTiles}
```

Comment:
Since you're using `<li>` in the map I would house your `{pizzaTiles}` in a `<ul>`